### PR TITLE
Ability to override helm deployment values

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Added the ability to override helm deployment values.

### How to review 
Go to the concourse PR https://github.com/ONSdigital/census-eq-concourse/pull/27 and use this as your branch for eq-survey-runner

### CODE REVIEW

File: k8s/helm/values.yaml [hunk 1]
```diff
@@ -26,6 +26,6 @@ resources:
     cpu: "3"
 
 autoscaler:
-    minReplicas: 3
-    maxReplicas: 10
+    minReplicas: ""
+    maxReplicas: ""
     targetCPUUtilizationPercentage: 50
```
Comment (ajmaddaford) [inline k8s/helm/values.yaml]: It would be useful to make this configurable as well.

Comment (pricem14pc) [reply k8s/helm/values.yaml]: We thought that too but maybe in another PR.

File: k8s/deploy_app.sh [hunk 2]
```diff
@@ -9,6 +9,12 @@ fi
 
 DOCKER_REGISTRY="${DOCKER_REGISTRY:-eu.gcr.io/census-eq-ci}"
 IMAGE_TAG="${IMAGE_TAG:-latest}"
+REQUESTED_CPU="${REQUESTED_CPU:-3}"
+MAX_UNAVAILABLE="${MAX_UNAVAILABLE:-1}"
+MAX_SURGE="${MAX_SURGE:-1}"
```
Comment (ajmaddaford) [inline k8s/deploy_app.sh]: It's not clear these are for rolling updates, it would be better to prefix the two with `ROLLING_UPDATE_`.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Comment by pricem14pc** [comment]
[Cluster min/max size](https://trello.com/c/RHIG9Sto/3369-cluster-min-max-size)

**Review by pricem14pc — APPROVED** [review]

**Pull request merged**